### DISCUSSION TITLE
Using a fork of `auto`

### DISCUSSION THREAD

**Discussion by MikeChongCan** [AbdBarho/stable-diffusion-webui-docker#407] [Ideas]
Re: <https://www.reddit.com/r/StableDiffusion/comments/12grgwh/automatic1111_getting_rusty_future_of_this_repo_i/>

https://github.com/vladmandic/automatic

Could we have an option to use the fork of `auto`?

**Comment by AbdBarho** [comment]
Seems like a good project, I would probably wait a couple of weeks to see where the project is headed.

**Reply by MikeChongCan** [reply]
Sure! Thank you!

**Comment by DevilaN** [comment]
Why are you thinking of replacing original when you can have both?

**Reply by AbdBarho** [reply]
hmmm, I wasn't actually thinking about replacing, but now that you say that, I think it replacing would make sense in the mid-/long run (assuming feature parity and continued maintenance).

**Reply by Kadah** [reply]
There's >11k forks, looks like around 100-200 active within the past week to some extent.

This fork appears to be broken on any OS and browser the maintainer isn't using.

**Comment by mckinlec** [comment]
There shouldn't be any issue with changing the AUTOMATIC1111 docker file on line 38  from "/AUTOMATIC1111/stable-diffusion-webui.git" to whatever fork you want right?

Unless it'll break something I'm not aware off

**Reply by AbdBarho** [reply]
In theory yes, in practice you have to check all mounts and configuration options, unless they are 100% compatible, which is unlikely.
